src/main/java/com/gargoylesoftware/htmlunit/WebClientUtil.java

```diff
@@ -136,5 +136,13 @@ public void malformedScriptURL(HtmlPage htmlPage, String url, MalformedURLExcept
         @Override
         public void loadScriptError(HtmlPage htmlPage, URL scriptUrl, Exception exception) {
         }
+
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public void warn(String s, String s1, int i, String s2, int i1) {
+
+        }
```

Comment:
I think it would make sense to use the implementation in https://github.com/jenkinsci/jenkins-test-harness/pull/179/files so the change matches better with the current coding style: no (meaningless) comment, correct parameter names, no empty line.

Reply:
take whichever, they should be pretty much the same PR now